frontend/src/api/k8s/__tests__/templates.spec.ts

```diff
@@ -59,32 +59,35 @@ const createServingRuntime = (name: string): K8sDSGResource => ({
 describe('assembleServingRuntimeTemplate', () => {
   it('should assemble serving runtime template with templateName', () => {
     const servingRuntimeMock = JSON.stringify(createServingRuntime('template-1'));
+    const servingRuntimeTemplatesMock = mockServingRuntimeTemplateK8sResource({
+      platforms: [ServingRuntimePlatform.MULTI],
+    });
+    delete servingRuntimeTemplatesMock.metadata.ownerReferences;
```

Comment:
@ppadti this `delete` should no longer be necessary right?

Reply:
yeah right, I did not updated the `mockServingRuntimeTemplateK8sResource` to remove `ownerReferences`. I will update the mock as well.